packages/composites/anchor/README.md

```diff
@@ -6,12 +6,14 @@ This composite adapter fetches the price of a given token from the Anchor protoc
-|    ✅     |            `RPC_URL`            |   The RPC URL to connect to the chain    |                                                                                                 |                                                |
-|           | `ANCHOR_VAULT_CONTRACT_ADDRESS` | The address of the Anchor Vault contract | Address can be found [here](https://docs.anchorprotocol.com/smart-contracts/deployed-contracts) |  `0xA2F987A546D4CD1c607Ee8141276876C26b72Bdf`  |
-|           | `TERRA_BLUNA_CONTRACT_ADDRESS`  |  The address of bLuna contract in Terra  | Address can be found [here](https://docs.anchorprotocol.com/smart-contracts/deployed-contracts) | `terra1mtwph2juhj0rvjz7dy92gvl6xvukaxu8rfv8ts` |
-|           |  `STETH_POOL_CONTRACT_ADDRESS`  |      The address of stEth contract       |                   Address can be found by finding the stETH/ETH pool in Curve                   |  `0xdc24316b9ae028f1497c275eb9192a3ea0f67022`  |
+| Required? |                Name                |               Description                |                                             Options                                             |                  Defaults to                   |
+| :-------: | :--------------------------------: | :--------------------------------------: | :---------------------------------------------------------------------------------------------: | :--------------------------------------------: |
+|    ✅     |         `ETHEREUM_RPC_URL`         |   The RPC URL to connect to the chain    |                                                                                                 |                                                |
+|           |  `ANCHOR_VAULT_CONTRACT_ADDRESS`   | The address of the Anchor Vault contract | Address can be found [here](https://docs.anchorprotocol.com/smart-contracts/deployed-contracts) |  `0xA2F987A546D4CD1c607Ee8141276876C26b72Bdf`  |
+|           | `TERRA_BLUNA_HUB_CONTRACT_ADDRESS` |  The address of bLuna contract in Terra  | Address can be found [here](https://docs.anchorprotocol.com/smart-contracts/deployed-contracts) | `terra1mtwph2juhj0rvjz7dy92gvl6xvukaxu8rfv8ts` |
+|           |   `STETH_POOL_CONTRACT_ADDRESS`    |      The address of stEth contract       |                   Address can be found by finding the stETH/ETH pool in Curve                   |  `0xdc24316b9ae028f1497c275eb9192a3ea0f67022`  |
+|           |     `LUNA_TERRA_FEED_ADDRESS`      |     The Terra address of `LUNA` feed     |        Address can be found by here https://docs.chain.link/docs/terra/data-feeds-terra/        | `terra1u475ps69rmhpf4f4gx2pc74l7tlyu4hkj4wp9d` |
+|           |      `ETH_TERRA_FEED_ADDRESS`      |     The Terra address of `ETH` feed      |        Address can be found by here https://docs.chain.link/docs/terra/data-feeds-terra/        | `terra1dw5ex5g802vgrek3nzppwt29tfzlpa38ep97qy` |
 
 **Additional environment variables must be set according to the Token Allocation adapter.**
```

Comment:
This is gone now right?

Reply:
I assume we should leave these here even with the defaults, since they may be at different addresses on different chains.

Reply:
I think Justin is talking about the reference to the TA adapter.  Yes this can be removed.